mate-settings-daemon/mate-settings-manager.c

```diff
@@ -215,21 +228,21 @@ _load_file (MateSettingsManager *manager,
                                   DEFAULT_SETTINGS_PREFIX,
                                   mate_settings_plugin_info_get_location (info));
 
-	/* Ignore unknown schemas or else we'll assert */
-	if (is_schema (schema)) {
-	       manager->priv->plugins = g_slist_prepend (manager->priv->plugins,
-		                                         g_object_ref (info));
+        /* Ignore unknown schemas or else we'll assert */
+        if (is_schema (schema)) {
+                manager->priv->plugins = g_slist_prepend (manager->priv->plugins,
+                                                          g_object_ref (info));
 
-	       g_signal_connect (info, "activated",
-		                 G_CALLBACK (on_plugin_activated), manager);
-	       g_signal_connect (info, "deactivated",
-		                 G_CALLBACK (on_plugin_deactivated), manager);
+                g_signal_connect (info, "activated",
+                                G_CALLBACK (on_plugin_activated), manager);
+                g_signal_connect (info, "deactivated",
+                                G_CALLBACK (on_plugin_deactivated), manager);
 
-	       /* Also sets priority for plugins */
-	       mate_settings_plugin_info_set_schema (info, schema);
-	} else {
-	       g_warning ("Ignoring unknown module '%s'", schema);
-	}
+                /* Also sets priority for plugins */
+                mate_settings_plugin_info_set_schema (info, schema);
+        } else {
+                g_warning ("Ignoring unknown module '%s'", schema);
+        }
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.